CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### [0.24.6](https://github.com/plus3it/tardigrade-ci/releases/tag/0.24.6)
+
+**Released**: 2023.02.15
+
+**Summary**:
+
+* Updates .pylintrc to specify module name for overgeneral-exceptions
```

Comment:
Needs to mention any tool versions that are included in this release. See prior release for an example. Here's the comparison link:

* https://github.com/plus3it/tardigrade-ci/compare/0.24.5...master

If you use the "Files changed" tab, it's pretty easy to see all what's updated.

Reply:
Updated